Bio/Nexus/Trees.py

```diff
@@ -13,10 +13,10 @@
 nodes).
 """
 
-
 import random
 import sys
 from . import Nodes
+import re
```

Comment:
Move between ``import random`` and ``import sys`` since these are already alphabetical (not something we're currently enforcing as a style rule, but might do one day).